docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1062,11 +1063,11 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  We are effectively registering auth plugin *_plugin identifier_* as a part of *_authentication category_*.
```

Comment:
*Auth plugin *_plugin identifier_* is being registered as a part of *_authentication category_*.